### PR TITLE
[WIP] Create database configuration state

### PR DESCRIPTION
**Pull request by bisht-richa**
Create database configuration page UI

### CODE REVIEW

File: assets/app/features/upgrade/i18n/en.json [hunk 1]
```diff
@@ -119,6 +119,19 @@
...
+      "database-configuration": {
+        "name": "",
+        "title": "Create or Connect to an OpenStack PostgreSQL Database",
+        "form": {
+          "username": "Username",
+          "password": "Password",
+          "create-database": "Create New Database",
+          "server": "server",
+          "port": "port",
+          "table-prefix": "Table_Prefix",
```
Comment (santiph) [inline assets/app/features/upgrade/i18n/en.json]: Please keep using capital first letters and space, not underscores for consistency across all the labels.

File: assets/app/features/upgrade/templates/upgrade7/database-configuration.jade [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+uib-tabset(active='active')
+  uib-tab(index='0', heading='Create a Database')
+    form
+      .form-group
+        label(translate='') upgrade7.steps.database-configuration.form.username
+        input.form-control(type='text', required='', ng-model='username')
+      .form-group
+        label(translate='') upgrade7.steps.database-configuration.form.password
+        input.form-control(type='text', required='', ng-model='model.password')
```
Comment (santiph) [inline assets/app/features/upgrade/templates/upgrade7/database-configuration.jade]: Good try to use ngModels, @bisht-richa! :)
Now we would need to create a new model in the controller's view model to map and handle those models. Something like

```
vm.form = {
    username: '',
    password: '',
    server: '',
    port: '',
    table_prefix: ''
}
```

Then in this view, we'd just bind them as `ng-model='vm.form.server'`

Please, let me know if you need help with it.

File: assets/app/features/upgrade/templates/upgrade7/database-configuration.jade [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        label(translate='') upgrade7.steps.database-configuration.form.username
+        input.form-control(type='text', required='', ng-model='username')
+      .form-group
+        label(translate='') upgrade7.steps.database-configuration.form.password
+        input.form-control(type='text', required='', ng-model='password')
+      .form-group
+        label(translate='') upgrade7.steps.database-configuration.form.table-prefix
+        input.form-control(type='text', required='', ng-model='able-prefix')
+
+      button.btn.btn-success.pull-right.btn-lg(translate='') upgrade7.steps.database-configuration.form.connect-database
```
Comment (santiph) [inline assets/app/features/upgrade/templates/upgrade7/database-configuration.jade]: Please, add a new line at the end of the file. ;)

**Comment by bisht-richa** [comment]
Everything is ok.

**Pull request merged**